### PR TITLE
Added GetExtents method

### PR DESCRIPTION
**Pull request by jo-asplin-met-no**

### CODE REVIEW

File: datastore/storagebackend/postgresql/getextents.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		Start: timestamppb.New(start),
+		End:   timestamppb.New(end),
+	}, nil
+}
+
+// getGeoExtent gets the current geo extent of all observations in the storage.
+func getGeoExtent(db *sql.DB) (*datastore.BoundingBox, error) {
+	query := `
+		SELECT ST_XMin(ext), ST_YMin(ext), ST_XMax(ext), ST_YMax(ext)
+		FROM (SELECT ST_Extent(point::geometry) AS ext FROM geo_point) t
```
Comment (lukas-phaf) [inline datastore/storagebackend/postgresql/getextents.go:35]: Nice way to do this!

File: datastore/protobuf/datastore.proto [hunk 2]
```diff
@@ -127,9 +135,21 @@ message GetObsRequest {
...
+//---------------------------------------------------------------------------
+
+message GetExtentsRequest {
+  // currently no args
+}
+
+message GetExtentsResponse {
+  int32 status = 1;
+  string error = 2; // any error description (empty on success)
+  TimeInterval time_extent = 3;
```
[the remark this answers is not in the document]
Comment (jo-asplin-met-no) [reply datastore/protobuf/datastore.proto:153]: Definitely! No problems with standardizing on this. Will do.

File: datastore/storagebackend/postgresql/getextents.go [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+	"fmt"
+	"time"
+
+	_ "github.com/lib/pq"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+// getTimeExtent gets the current time extent of all observations in the storage.
+func getTimeExtent(db *sql.DB) (*datastore.TimeInterval, error) {
+	query := "SELECT min(obstime_instant), max(obstime_instant) FROM observation"
```
[the remark this answers is not in the document]
Comment (jo-asplin-met-no) [reply datastore/storagebackend/postgresql/getextents.go:15]: I suggest we wait a bit until the API stabilizes and we have time to do extensive performance testing (which is something we need for other use cases / methods as well).

File: datastore/storagebackend/postgresql/getextents.go (hunk 1 above)
Comment (jo-asplin-met-no) [reply datastore/storagebackend/postgresql/getextents.go:35]: 🙂 Not sure if this is the best idiom (wrt. performance etc.) in this case, but it looks nice, I agree.

**Pull request merged**